extension/sidebar-pane.html

```diff
@@ -127,6 +127,53 @@
       font-weight: 600;
     }
 
+    /* occurrences part */
+    .occurrences header {
+      display: flex;
+      align-items: center;
+      cursor: pointer;
+      color: #b6babf;
+    }
+
+    .occurrences header::before {
```

Comment:
If using a `<details>` HTML element, this pseudo-element and its `-90deg` rotation for "collapsed" state won't be needed.